front/src/reducers/osrdconf/osrdConfCommon/__tests__/utils.ts

```diff
@@ -1,14 +1,13 @@
 import { omit } from 'lodash';
 import { createStoreWithoutMiddleware } from 'store';
+import commonConfBuilder from 'reducers/osrdconf/osrdConfCommon/__tests__/commonConfBuilder';
```

Comment:
Better to keep all reducers imports together like you did for the other files

Reply:
done